src/main/anovos/data_analyzer/association_evaluator.py

```diff
@@ -146,17 +147,15 @@ def variable_clustering(
     idf,
     list_of_cols="all",
     drop_cols=[],
-    sample_size=100000,
     stats_mode={},
     print_impact=False,
 ):
     """
     Variable Clustering groups attributes that are as correlated as possible among themselves within a cluster and
-    as uncorrelated as possible with attribute in other clusters. The function is leveraging [VarClusHi] [2] library
-    to do variable clustering; however, this library is not implemented in a scalable manner due to which the
-    analysis is done on a sample dataset. Further, it is found to be a bit computational expensive especially when
-    number of columns in the input dataset is on higher side (number of pairs to analyse increases exponentially with
-    number of columns).
+    as uncorrelated as possible with attribute in other clusters. The function is leveraging VarClusHiSpark class from
+    anovos.data_analyzer.association_eval_varclus file, which is a scalable version of [VarClusHi] [2] library
+    to do variable clustering. The VarClusHiSpark is implemented in a scalble manner with necessary optimizations,
+    thus sampling is not required.
 
     [2]: https://github.com/jingtt/varclushi   "VarCluShi"
```

Comment:
We can move this, and the explanation for "scalable version of VarClusHi" inside our `VarClusHiSpark` as stated above, and only focus on briefly walking through the algorithm and the wrapper logic here.